python/create_graph_data.py

```diff
@@ -22,7 +22,7 @@ def get_recent_escapement(hatchery_provider_response: HatcheryProviderResponse):
     filtered_df = df[df["date"] > two_weeks_prior]
 
     if len(filtered_df) == 0:
-        return []
+        return json.dumps([])
```

Comment:
Small fix - We might be missing some data in S3 rn but after this deploys the data will start populating when the job runs